src/filters/birds.js

```diff
@@ -58,20 +75,58 @@ const active = (collection, current = true) =>
   );
 
 /* @docs
-label: authorPage
-category: Pages
-note: Return the home page of a given author
+label: activeAuthor
+category: Status
+note: |
+  Check if any author of a page is currently active
+  (if 'oddbird' is an author, this will returrn true)
 example: |
-  {{ collections.all | authorPage('miriam') }}
+  {{ post | activeAuthor(collections.all) }}
+params:
+  page:
+    type: 11ty page object
+  all:
+    type: array
+    note: |
+      Any collection that includes the author pages,
+      such as `collections.all`
+*/
+const activeAuthor = (page, all) => {
+  const author = page.data.author || [];
+  const list = typeof author === 'string' ? [author] : author;
+  return (
+    list.includes('oddbird') ||
+    list.some((name) => isCurrent(authorPage(all, name)))
+  );
+};
+
+/* @docs
+label: activeAuthor
+category: Status
+note: |
+  Filter a collection to only include
+  pages with a currently active author.
+  (if 'oddbird' is an author, this will returrn true)
+example: |
+  {{ collections.oss | withActiveAuthor(collections.all) }}
 params:
   collection:
+    type: 11ty collection
+    note: |
+      The collection to filter
+  all:
     type: array
-    note: containing 11ty page objects
-  bird:
-    type: string
-    note: The name of the bird (as used in `author` settings)
+    note: |
+      Any collection that includes the author pages,
+      such as `collections.all`
 */
-const authorPage = (collection, bird) =>
-  withData(collection, 'data.bird', bird)[0];
+const withActiveAuthor = (collection, all) =>
+  collection.filter((page) => activeAuthor(page, all));
```

Comment:
Conversely, do we want to exclude `oddbird`-owned OSS projects from inactive author detail pages?

Reply:
I buy that. That would be a different filter, right?

Reply:
(the only ex-bird with a detail page currently is David…)

Reply:
https://trello.com/c/nHKy1mKo/59-dont-need-to-show-oddbird-projects-on-emeritus-bird-pages